cfgov/mega_menu/wagtail_hooks.py

```diff
@@ -54,7 +59,7 @@ def get_admin_urls_for_registration(self):
         urls = super().get_admin_urls_for_registration()
 
         return urls + (
-            url(
+            path(
                 self.url_helper.get_action_url_pattern('preview'),
```

Comment:
Does `self.url_helper.get_action_url_pattern('preview')` return a `path` string and not a regular expression?

Reply:
I changed `path` to `re_path` to get rid of the WARNING messages logged for `urlpatterns` in `mega_menu` and `regulations3k`